src/features/landing/components/AuthSection/AuthForm/AuthLayout.tsx

```
@@ -0,0 +1,100 @@
+  const triggerFormSubmit: () => void = () => {
+    if (formRef.current?.submit) {
+      formRef.current.submit();
+    }
+  };
+
+  return (
+    <Box sx={styles.formWrapper}>
+      {loading && (
+        <Box
+          sx={styles.loader}
+          role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)